packages/flutter_tools/lib/runner.dart

```diff
@@ -62,19 +63,6 @@ Future<int> run(
     StackTrace? firstStackTrace;
     return runZoned<Future<int>>(() async {
       try {
-        // Ensure that the consent message has been displayed
```

Comment:
Do we need to also cherrypick package:unified_analytics to get this functionality, or is it already in version 1.0.1?

Reply:
Oh I didn't realize that the pubspec's would also need to be manually cherrypicked, but yes, we will need the latest `1.1.0` in the pubspec for this functionality. What's the approach for bumping the package version? Cherry pick the engine roll commit or can I just bump it in this cherrypick?